MpsAllocatorSample/Program.cs

```diff
@@ -203,6 +203,17 @@ static async Task RequestMultiplayerServer()
             string region = Console.ReadLine();
             req2.PreferredRegions = new List<string>() {region};
             req2.SessionId = Guid.NewGuid().ToString();
+            // Initial list of players (potentially matchmade) allowed to connect to the game.
+            // This list is passed to the game server when requested (via GSDK) and can be used to validate players connecting to it.
+            Console.WriteLine("Type initial players ID, Enter to confirm. Empty string when you're done");
```

Comment:
Should we say PlayFab EntityId instead of players "Id"

Reply:
This is dependent on the game, we don't require game devs to use PlayFab auth for their game servers.